runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkRunner.java

```diff
@@ -126,7 +133,43 @@ static PipelineResult createPipelineResult(JobExecutionResult result, PipelineOp
       if (accumulators != null && !accumulators.isEmpty()) {
         LOG.info("Final accumulator values:");
         for (Map.Entry<String, Object> entry : result.getAllAccumulatorResults().entrySet()) {
-          LOG.info("{} : {}", entry.getKey(), entry.getValue());
+          MetricsContainerStepMap stepMap = (MetricsContainerStepMap) entry.getValue();
```

Comment:
I believe `MetricsContainerStepMap` was just being output as an object address here, so I expanded it to something more meaningful.

This could get factored out to a helper (overriding `MetricsContainerStepMap.toString` to do this feels like too much)